src/identity.d.ts

```diff
@@ -353,6 +353,76 @@ export type LoginOptions = {
      * in place of default teaser
      */
     teaser?: string;
+    /**
+     * - Specifies the allowable elapsed time in seconds since
+     * the last time the End-User was actively authenticated. If last authentication time is more
+     * than maxAge seconds in the past, re-authentication will be required. See the OpenID Connect
+     * spec section 3.1.2.1 for more information
+     */
+    maxAge?: number | string;
+    /**
+     * - Optional parameter to overwrite client locale setting.
+     * New flows supports nb_NO, fi_FI, sv_SE, en_US
+     */
+    locale?: string;
+    /**
+     * - display username and password on one screen
+     */
+    oneStepLogin?: boolean;
+    /**
+     * - String that specifies whether the Authorization Server prompts the
+     * End-User for reauthentication or confirm account screen. Supported values: `select_account` or `login`
+     */
+    prompt?: string;
+};
+export type SimplifiedLoginWidgetLoginOptions = {
```

Comment:
I *think* you express this as

```ts
export type SimplifiedLoginWidgetLoginOptions = Omit<LoginOptions, 'state'> & {
    /**
     * - An opaque value used by the client to maintain state between
     * the request and callback. It's also recommended to prevent CSRF {@link https://tools.ietf.org/html/rfc6749#section-10.12}
     */
    state: string | (() => Promise<string>) | (() => string);
};
```
in order to reuse the options but redefine the `state`-property.

[TS Playground](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBDAnmYcAyEDmBLAdgeTBmwlwGc4BeOAbwCg5G4B6AKlYablbgFo4AgrjgQwAQwCOAV1QA3MQBsZcKWWAATOACNECABaoAxguzBc8GBDgBbMXhh3hZBzFRbgMAO7AznJjxgDOChgaWBnODFcTUNFBS0xQwBrADo4AEkYAHIKRTIrEMMIa2szdQ0EKzAQ2TN4AGEAZQAlADFaAAETXCS4PRgYMDIALmZmSwgFMhTTGAAzFOhMZn7rBWYoOcMANgB2ABYATgBiNUNiUl4ARgAGFKuAJgBfP0ZWZlfnMVdhuGcoPCYADcrzYHC43D4gikgTq2FiF2E9VIrhADQUYjIFGawDmwBCuEMqAAaooZNMMnNXpDitgBhoADT6VCqfFwTzYBQKbSoTFJCqWSIwgzmeHfFlkQEqNRQXBiUoAanEWM80HUKWpPFa0DgD1aAjgAAoACqq3itRKWKDQ2GihEkXAASjgkCx2C0CjkZPCkRCcAABmRrGR-Uz-RBBv6jaRUMRSi7MWRVVB1M6oupNQHlUm1VHDRyuXA5tAiQmVWq4EVcHNsFBbIimcBasJsHNpWzsLkFCExOpdAosJgKnhHWljXoIGo4PIlD7rNhMP1M+4bNgQBVMX9xETeGpxFBxZp-oCx1ZbPy-lI-YFvu3rXpN2JhXCERUOYFdfq4IgIFI4LgfAzCEAisWoATmXQBAAWWaI0BGfO1vgdOAoI8Cd1GxXF8TMIkyGdYw7GsOA8AyAARSp+VwDVgJQhd+kiKYrBXVlNEFMwyCvXl1HUOkHUUSJDGtQ0gzIJkIzAZ1i2tRQez7OAB0wIdNBI1koGmTNGikMBIFgCoZ3JX1eUYuAsnE3hgFsTkshsYAogoGMEGweNs2TTRVClCy7AUJl0xMsyROs0o7MzBy41QFyK3c3BMD+YNqK4d5XkSKBSVnMgAH5fmPaKQS4MFM34cdUHweCPzIIoUAoKTmQo9ixz0TtiNyeTO3gCAqRo8rRHCJk9zEA9XE0HQt0SYA0nSNtYRaiJ2r+CqfSrBw8AocMUFwbB1FDGqhwAgaBQgSiyEzPBjCkcoKCmjbauETwGsMPRKyiHl2zmKRuWqocBilPApPrZCxC0X8LCCVS0nwecvuizMupQJqGIHbx1CZW7OVQf08BnDaAH0YeAKNGpCGArwAzRbrMGrjFMcxIjIJIqp1MQ5u679f0zfqzEATAJ4DiCBEcqYJQnJGA0hoLo8F6fpBhGMYJimGYPAWJYVhgNYNi2PYjlOYBzgdXgAGYUj1l5gI+Lhccyv4YABHLQXYAq4BxHjCngK9sH0W8C25QpgGwWpmUzIpyjSKDVHgUBLQUXR63uyIBad7WYAAVQBIsoGKFmr0rEw6mIpxgAUOZd3xWR4WATVTaYEJ4-OZPsAt7LgVt8EEqhRoJze0nYygXRBW64RGcgMAtPZPB1F59Ly9eaosKgAAFUQtItwHJls3BcqYfKaP4VS4E8zkRGtRPE-SciGvMSeuAUvAAAl7Hrq3AXXxhN5b-hZ7eqcHEwC+mC-+-rcbnlO2W84DGlsjKP4ShMBjnAWyd8D0cDNkgVIGKntno8TIGADEiANDHWEFg0aIg2zlDmE+BQFhYFQB-owVwmJ8T-0fk3e2jQUCGFbKYC6QQeaeABp6XeGIhgCicqgEiZxSAYT+CdMuNEpoYgiGFGqABRaIvBE4QJ4bkHWtRI5CltMQV86pKTyUxNzBC+ikKkEcvGRq1hoDSJbjeYQtgQACCHH8bW4iKAkSmsqGATIQi8CfHosUiIR6FhXCEaQtYNBpEaD4GqhAzAnzgMiXAAFzjQ1Ye4nWliDZXBSA8e4RYdS2L9D9aAf1SDUJsGIFxQ4La4CkNYdw1oAA+lsAFPxYMA1+cBCCIn4vueUHg2S9zAp4AErgs5U3gAOWIfC1CQ2gZmAAcsATwRYEYUA4tpaAMAKC4C0FjFZ+AmQ1ixq0dIvVZBY0aIoxsuAsaJ0aNUuZihgAMJtkA5u-goQYIIboVScp4y+QiimEQwgHLlRCL4E2rwYyNFcGADAOBcBLwgCvKIXSX6-P4IigB7t4CYO1uwn0ZNYTWimqVCcAIABeFjhBxKgGBF0adrBEE4fY35yj1CqIgdVHsZiQnIR1FWGsdYBJFCkNTaFPgqJwE0rs3Smh9LhF+IGfOCcsaJCleYKMOp-RX1wP6ap1RihEE+Y3J4IJQA6QsMgVAjQnJYNJeoFFeAADqG1PruoIEQB0FBqDgzpAAHl9f0gNTIcguGAFkAAfHAAAZLQJhIChAiHEGEac3p2xDR7kESmOdBSWXMI4S24oeReDlZmKakShaRGiI9LkCRkjjWyF2fIAsiglDKPtVlTYc5NDaJ0boEsBhDFGOMDFctZiKygMsVY6xNg7AOCcM4iJrh3EeMbBKFdGBfB+B0qU7TDSGmdJQBNs82WdmACGhucbnQnrPVQBNDdHQgmtXQOgVZ5HhBgL8J17KTA1g0L6r16gfVYDwBG0ggaU1cExIgQk5bXDPvoBCSuHhiYmVcM4LIXSXgvG-bBihzgHgAedcB0wbqoO4HA5B1FMHyBUHg0wA9wBfhZFw8QaKWQ6DWqAA)

Reply:
While that is valid for TS, it is not for JSDoc, and since this is generated from that it get's a bit harder